components/admin_console/schema_admin_settings.jsx

```diff
@@ -859,15 +860,13 @@ export default class SchemaAdminSettings extends React.Component {
         return Boolean(SchemaAdminSettings.getConfigValue(this.props.environmentConfig, path));
     };
 
-    render = () => {
-        const schema = this.props.schema;
+    customComponentWrapperClass = (className) => {
```

Comment:
All this callback thing to change the class of the sibling component looks a bit weird for me. can we look for a cleaner way to solve this?

Reply:
The issue was that `.form-horizontal` in `renderSettingsWrapper` function causes a large gap between the top of the page and the user activity log table. The sibling component, `CustomComponent` wraps `audits.jsx` in this particular case, so I needed to somehow tell `renderSettingsWrapper` to use different styling only when specified. Because of the lack of hierarchy between these elements, I couldn't specify a fix using CSS only. It's the only workaround I could think of that didn't involve changing the CSS that may affect the whole system console.